### PR TITLE
feat: Revert 1.11.0

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

1.11.0 has been unpublished, so revert the commit.

https://github.com/eggjs/egg-bin/pull/51#issuecomment-299775338

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "egg-utils": "^1.0.0",
     "glob": "^7.1.1",
     "intelli-espower-loader": "^1.0.1",
-    "istanbul": "^1.1.0-alpha.1",
```
Comment (atian25) [inline package.json]: 咦，眼花了？我怎么记得之前用的是 `1.0.0-alpha.1` 很久了

Comment (popomore) [inline package.json]: 是 master 用的

Comment (atian25) [inline package.json]: 喔喔，这是 1.x 的

**Comment by popomore** [comment]
+ egg-bin@1.11.1

**Comment by fengmk2** [comment]
也好，发现了 cnpmjs.org 新接口的bug

**Comment by fengmk2** [comment]
https://github.com/cnpm/cnpmjs.org/pull/1192

**Pull request merged**